raster/r.surf.gauss/main.c

```diff
@@ -55,9 +57,32 @@ int main(int argc, char *argv[])
     sigma->type = TYPE_DOUBLE;
     sigma->answer = "1.0";
 
+    seed = G_define_option();
+    seed->key = "seed";
+    seed->type = TYPE_INTEGER;
+    seed->required = NO;
+    seed->label = _("Seed for random number generator");
+    seed->description = _("The same seed can be used to obtain same results"
+                          " or random seed can be generated by other means.");
+
```

Comment:
Do we have standardized messages for common options between modules?

Reply:
Not really, we would have to add a new standard option (something like G_OPT_M_SEED) to parser. That would make sense, but maybe as a separate PR?